runtime/tr.source/trj9/build/platform/host/s390-ztpf64-gcc.mk

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2000, 2017 IBM Corp. and others
```

Comment:
Is this initial date correct?  The code here dates back to the 2000?

Reply:
Thanks, since I've only just created the file, there's no need for two dates in the copyright statement.